script/Arcueid/Day 10/10_00_ARC10_5E.txt

```diff
@@ -591,7 +591,7 @@ I didn't stay together with Arcueid because of anyone else's will.
 [sha:43104bd79f51b589d87c1295ee301a99240b1dc8]{
 -- Page 666, Offset 26043.
 -- 「……何を言っているんです？　一度も血を吸った事がない、ですって？　そんな戯言を信じろとでも？
-"...What are you on about? She's never sucked anyone's blood? You expect us to believe that bullshit?
+"...What are you on about? She's never sucked anyone's blood? You expect us to believe that baloney?
```

Comment:
Hmm it looked off when Noel said it seriously.

Reply:
Sorry but I cannot take baloney seriously. Cripes moment.